system/Common.php

```diff
@@ -818,7 +818,7 @@ function redirect(string $uri = null)
 
 		if (! empty($uri))
 		{
-			return $response->route($uri);
```

Comment:
This was originally forced to use `route` instead of `to` to make it difficult to redirect to pages outside the site on accident. This was inspired by OWASP's 2013 Top 10 list, [Unvalidated Redirects and Forwards](https://www.owasp.org/index.php/Top_10_2013-A10-Unvalidated_Redirects_and_Forwards). It is no longer in the Top 10 list, but still a decent security precaution, I believe.